crypto/fipsmodule/aes/asm/aes-gcm-avx2-x86_64.pl

```diff
@@ -958,6 +958,73 @@ sub _aes_gcm_update {
 $code .= _aes_gcm_update 0;
 $code .= _end_func;
 
-print $code;
+sub filter_and_print {
+    # This function replaces AVX2 assembly instructions with their assembled forms,
```

Comment:
Technically these are VAES and VPCLMULQDQ instructions, not AVX2.

Reply:
@ebiggers Thanks. I really appreciate the attention to detail. I clarified this in PR #2480.